examples/with-react-native-web/packages/tsconfig/react-native-app.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "React Native App",
+  "extends": "./base.json",
```

Comment:
Can this line extend `expo/tsconfig.base` without making the `tsconfig` package depend on the entire world of metro?